cmd/gcp-controller-manager/node_annotator.go

```diff
@@ -126,6 +127,10 @@ func newNodeAnnotator(client clientset.Interface, nodeInformer coreinformers.Nod
 						return false
 					}
 
+					if resizeRequestName := extractResizeRequestLabel(instance); resizeRequestName != nil {
```

Comment:
Can we do this in an independent annotator (e.g. "provisining-request-reconciler")? While genericly enough named, this loop is for node labels set via gcloud container clusters create --node-labels.

Reply:
Done, introduced new annotator `resize-request-reconciler`.